dev-python/frozendict/frozendict-2.3.4.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A simple immutable mapping for python"
+HOMEPAGE="
+	https://github.com/Marco-Sulla/python-frozendict
+	https://pypi.python.org/pypi/frozendict
```

Comment:
Missing slash. `pkgcheck scan --net` is your friend.

Reply:
Thanks! I did not know this one.

Reply:
I mean the `--net` parameter :)